tree/20_cfe_basics/flag_lib.cf

```diff
@@ -169,16 +198,17 @@ bundle agent set_flag(name) {
       "notreserved"
         bundle_return_value_index => "1",
                           comment => "${flag_unique.value}",
-                               if => "!flag_reserved_${cname}";
+                               if => "!reserved";
```

Comment:
notreserved is never used - is it for future usage ?

Reply:
It is actually used for more precise testing (and maybe for future usage too).

Reply:
Great ! Thank you